### DISCUSSION TITLE
First Time User - how to run

### DISCUSSION THREAD

**Discussion by arturobandissimo** [ep1cman/unifi-protect-backup#177] [General]
What is the best way to run this if I don't already have a server-computer set up on my network?

I have a synology drive, unifi cloudkey, USG, etc.

Could I use the synology drive to run it?

**Comment by ep1cman** [comment]
https://www.reddit.com/r/Ubiquiti/s/L4LdeT1ogh < Someone has posted this for running it on synology, but I can't verify it as I don't have one.

**Reply by arturobandissimo** [reply]
TYSM
